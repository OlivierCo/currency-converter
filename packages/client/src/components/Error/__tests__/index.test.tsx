import React from 'react'
import { shallow } from 'enzyme'
import { Error } from '../index'

describe('Error', () => {
  it('should be defined', () => {
    expect(Error).toBeDefined()
  })

  it('should render correctly', () => {
    const wrapper = shallow(<Error />)

    expect(wrapper).toMatchSnapshot()
  })
})
