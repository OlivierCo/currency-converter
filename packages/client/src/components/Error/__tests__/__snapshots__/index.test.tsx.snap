// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly 1`] = `
<Container
  center={true}
>
  <EmotionCssPropInternal
    __EMOTION_LABEL_PLEASE_DO_NOT_USE__="Error"
    __EMOTION_TYPE_PLEASE_DO_NOT_USE__={[Function]}
    align="center"
    css={
      Object {
        "map": undefined,
        "name": "10klw3m",
        "next": undefined,
        "styles": "height:100%;",
        "toString": [Function],
      }
    }
    justify="center"
  >
    <span>
      Something went wrong I'm afraid...
    </span>
  </EmotionCssPropInternal>
</Container>
`;
